front/src/applications/editor/Editor.scss

```diff
@@ -181,18 +181,33 @@
       }
     }
   }
-}
 
-.z-index-tooltip {
-  z-index: 1070 !important;
-}
+  .selection-left-panel {
+    .entity {
+      .strong {
+        font-weight: bold;
+      }
+    }
+  }
 
-.editor-infra-errors-list {
-  .error-count {
-    height: 1.5rem;
+  .popup.editor-selection {
+    .maplibregl-popup-content {
+      min-width: 200px;
+      min-height: 100px;
+    }
   }
-  #errors-list-container {
-    height: 60vh;
-    overflow: auto;
+
+  .z-index-tooltip {
+    z-index: 1070 !important;
+  }
```

Comment:
Not sure this needs to be in this css file as its linked to the navbar. But it doesn't concern this PR :)

Reply:
Yes I agree :) there is already this refacto ticket https://github.com/osrd-project/osrd/issues/5421 about z index